FIXME: in theory this *should* work but it clearly is not.

--- origsrc/dbus-1.10.8/configure.ac	2016-03-07 10:06:22.000000000 -0600
+++ src/dbus-1.10.8/configure.ac	2016-05-06 14:25:35.816044900 -0500
@@ -137,7 +137,7 @@ fi
 
 # For best security, assume that all non-Windows platforms can do
 # credentials-passing.
-AS_IF([test "$dbus_win" = yes],
+AS_IF([test "$dbus_cygwin$dbus_win" != nono],
     [DBUS_SESSION_CONF_MAYBE_AUTH_EXTERNAL="<!--<auth>EXTERNAL</auth>-->"],
     [DBUS_SESSION_CONF_MAYBE_AUTH_EXTERNAL="<auth>EXTERNAL</auth>"])
 AC_SUBST([DBUS_SESSION_CONF_MAYBE_AUTH_EXTERNAL])
--- origsrc/dbus-1.10.12/bus/system.conf.in	2015-10-22 12:34:22.000000000 -0500
+++ src/dbus-1.10.12/bus/system.conf.in	2016-10-10 13:32:37.353222600 -0500
@@ -32,8 +32,15 @@
   <!-- Enable logging to syslog -->
   <syslog/>
 
-  <!-- Only allow socket-credentials-based authentication -->
-  <auth>EXTERNAL</auth>
+  <!-- On Unix systems, the most secure authentication mechanism is
+  EXTERNAL, which uses credential-passing over Unix sockets.
+
+  This authentication mechanism is not available on Windows,
+  is not suitable for use with the tcp: or nonce-tcp: transports,
+  and will not work on obscure flavours of Unix that do not have
+  a supported credentials-passing mechanism. On those platforms/transports,
+  comment out the <auth> element to allow fallback to DBUS_COOKIE_SHA1. -->
+  @DBUS_SESSION_CONF_MAYBE_AUTH_EXTERNAL@
 
   <!-- Only listen on a local socket. (abstract=/path/to/socket 
        means use abstract namespace, don't really create filesystem 
